autoload/UltiSnips/bootstrap.vim

```diff
@@ -20,10 +22,18 @@ function! UltiSnips#bootstrap#Bootstrap()
        endif
        let g:UltiSnipsUsePythonVersion = "<tab>"
    else
-       if g:UltiSnipsUsePythonVersion == 2
-           let g:_uspy=":py "
-       else
-           let g:_uspy=":py3 "
+       " Use user-provided value, but check if it's available.
```

Comment:
It uses `g:UltiSnipsUsePythonVersion`, but still checks if it's available in Vim.
Before this, you could set it to `3`, but `has('python3')` might have been false and then followup errors were happening.